amundsen-kube-helm/templates/helm/amundsen/templates/deployment.yaml

```diff
@@ -30,12 +46,24 @@ spec:
   selector:
     matchLabels:
       run: {{ .Chart.Name }}-{{ .Values.metadataServiceName }}
-  replicas: 1
+  replicas: {{ .Values.metadataReplicas }}
   template:
     metadata:
       labels:
         run: {{ .Chart.Name }}-{{ .Values.metadataServiceName }}
     spec:
+      {{- with default .Values.nodeSelector .Values.metadata.nodeSelector }}
```

Comment:
wdyt about wrapping the nodeSelector, affinity, tolerations sections in an if block, such that they are not required in all values files?

Reply:
Actually, it looks like `with default` will cause this to not be used, if neither nodeSelector is defined?

Reply:
Yeah. If both are set the `metadata.nodeSelector` one overrides the `nodeSelector`. If neither are set you don't get anything in the with clause.

Reply:
Cool, thanks TIL